.travis.yml

```diff
@@ -1,15 +1,20 @@
+dist: bionic
 language: go
+
 go:
-  - 1.9.x
-  - 1.10.x
-  - 1.11.x
+  - "1.13"
 
 branches:
   only:
-  - master
+    - master
+
+env:
+  global:
+    - GO111MODULE=on
+    - GOPROXY=https://goproxy.io
 
-install: make bootstrap
```

Comment:
I think the `bootstrap` should stay, since users on earlier versions of Go might not have go modules

Reply:
sounds good